draft-cenzano-moq-media-interop.md

```diff
@@ -264,6 +266,24 @@ It will be 0 if not set
 
 Opus packets, as described in {{!RFC6716}} - section 3
 
+#### UTF-8 Text
+
+~~~
+{
+  Seq ID (i)
+  Payload (..)
+}
+~~~
+{: #object-utf8-text format title="MOQT UTF-8 Text"}
+
+##### Seq Id
+
+Monotonically increasing counter for this media track
```

Comment:
Do you need this or can you just fall back to group ID + object ID?  I guess a global counter is useful for finding gaps around group boundaries.

Reply:
I need this perhaps the encoder dropped a frame / data, this is above transport layer